app/controllers/application_controller.rb

```diff
@@ -2,4 +2,15 @@ class ApplicationController < ActionController::Base
   # Prevent CSRF attacks by raising an exception.
   # For APIs, you may want to use :null_session instead.
   protect_from_forgery with: :exception
+  before_action :configure_permitted_parameters, if: :devise_controller?
+
+  def after_sign_out_path_for(resource)
```

Comment:
Unused method argument - `resource`. If it's necessary, use `_` or `_resource` as an argument name to indicate that it won't be used. You can also write as `after_sign_out_path_for(*)` if you want the method to accept any arguments but don't care about them.